demisto_sdk/commands/init/templates/BaseScript/BaseScript_test.py

```diff
@@ -9,7 +9,8 @@
 """
 
 import io
-import json
+from demisto_sdk.commands.common.handlers import JSON_Handler
```

Comment:
perhaps it'd be better import it at the top of the file, to avoid mistaken uses of other json implementations elsewhere in the file

Reply:
isort!